mmv1/products/vertexai/IndexEndpoint.yaml

```diff
@@ -65,7 +68,6 @@ parameters:
     url_param_only: true
     immutable: true
 properties:
-  # Intentionally deployedIndexes[] is not included because it's an output-only field and another terraform resource will manage a deployed index.
```

Comment:
Any reason we're removing this comment?

Reply:
Ah, I initially defied deployedIndexes in this PR, but I reverted it because of https://github.com/GoogleCloudPlatform/magic-modules/pull/8851#discussion_r1313955403. At that time, I forgot to revert the comment. I'll restore it. Thank you for the catch!

Reply:
@shuyama1 I fixed it in 40903e2 :+1: When you have time, could you approve CI again? Thanks